verticapy/__init__.py

```diff
@@ -46,7 +46,7 @@
 # data around for processing, VerticaPy brings the logic to the data.
 #
 #
-__version__ = "0.9.0"
+__version__ = "0.9.0-0"
```

Comment:
I think it was decided in the past that version of VertiaPy has 3 parts: MAJOR.MINOR.PATCH. So, it should be `0.9.1` instead of `0.9.0-0`.

Reply:
I know but we will not release it. It is mainly to avoid collision when a user is pulling the repo with the version 0.9.1 (not the real one but just the last merge), he will not be able to upgrade to 0.9.1 (when this version is released). This is why I am adding an extra digit.